packages/client/src/classes/client.js

```diff
@@ -15,26 +11,18 @@ const {
   },
 } = require('@sendgrid/helpers');
 
-/**
- * Twilio SendGrid REST Client
- */
+const API_KEY_PREFIX = 'SG.';
+
 class Client {
-  /**
-   * Constructor
-   */
   constructor() {
-
-    //API key
     this.apiKey = '';
 
-    //Default headers
     this.defaultHeaders = {
-      'Accept': 'application/json',
+      Accept: 'application/json',
```

Comment:
Why do we convert to a non-string key here? Why only for the `Accept` key?

Reply:
Quotes aren't needed here and is an eslint warning.